src/i18n/he.json

```diff
@@ -45,6 +45,9 @@
     "lang-control-text-ru": "Русский",
     "lang-control-text-en": "English"
   },
+  "mobile-menu": {
+    "label_to-main-menu": "חזור לתפריט הראשי"
```

Comment:
need to check how the new mobile interface works for rtl documentation

Reply:
Thanks, I checked and found a few bugs. I've already fixed it